addons/bus/websocket.py

```diff
@@ -782,14 +782,13 @@ def _serve_ir_websocket(self, event_name, data, context=None):
         appropriate ir.websocket method since only two events are
         tolerated: `subscribe` and `update_presence`.
         """
-        ir_websocket = self.env['ir.websocket']
-        ir_websocket._authenticate()
```

Comment:
note: `_authenticate`  is updating the `env`, but it updated the `env` of `ir_websocket` instead of `self`